modules/templates/SSF/views/index.html

```diff
@@ -74,7 +74,7 @@
    <div class='span4'>
     
     <h2><i class='icon-news'></i> {{=T("Updates Feed")}}</h2>
-    <div class = 'well'>
+    <div class = 'well-large'> /* made the box of news feed bigger */
```

Comment:
/* */ is not a comment in HTML, it is a comment in CSS. HTML comments are <!-- something -->